frontend/src/components/game/PeoplePicker.tsx

```diff
@@ -27,81 +27,74 @@ const Select: React.FC<PeoplePickerProps> = ({
-            stroke-width="2"
-            d="M19 13l-7 7-7-7m14-8l-7 7-7-7"
-          />
-          {/* rendering the "enemy" */}
-          <div className="">
-            {/* index 0 first*/}
-            <div className="">
-              {people[0] && (
-                <div
-                  key={people[0].id}
-                  onClick={() => disabled===false && onSelectPerson(people[0].id)}
-                >
-                  {avatar(
-                    people[0],
-                    `${people[0].name}`,
-                    people[0].id === selectedPerson,
-                    75,
-                    disabled
-                  )}
-                </div>
+    <div>
+      <path
+        stroke-linecap="round"
+        stroke-linejoin="round"
+        stroke-width="2"
+        d="M19 13l-7 7-7-7m14-8l-7 7-7-7"
+      />
+      {/* rendering the "enemy" */}
+      <div>
+        {/* index 0 first*/}
+        <div>
+          {people[0] && (
+            <div
+              key={people[0].id}
+              onClick={() => disabled === false && onSelectPerson(people[0].id)}
+            >
+              {avatar(
+                people[0],
+                `${people[0].name}`,
+                people[0].id === selectedPerson,
+                75,
+                disabled
               )}
             </div>
-            {/* index 1 second user*/}
-            <div className=" mt-6 grid grid-cols-2 col-span-1">
-              {people[1] && (
-                <div
-                  key={people[1].id}
-                  onClick={() => disabled===false && onSelectPerson(people[1].id)}
-                >
-                  {avatar(
-                    people[1],
-                    `${people[1].name}`,
-                    people[1].id === selectedPerson,
-                    50,
-                    disabled
-                  )}
-                </div>
+          )}
+        </div>
+        {/* index 1 second user*/}
+        <div className=" mt-2 grid grid-cols-2 col-span-1">
+          {people[1] && (
+            <div
+              key={people[1].id}
+              onClick={() => disabled === false && onSelectPerson(people[1].id)}
+            >
+              {avatar(
+                people[1],
+                `${people[1].name}`,
+                people[1].id === selectedPerson,
+                50,
+                disabled
               )}
-              {/* index 2: third user*/}
-              {people[2] && (
-                <div
-                  key={people[0].id}
-                  onClick={() => disabled===false && onSelectPerson(people[2].id)}
-                >
-                  {avatar(
-                    people[2],
-                    `${people[2].name}`,
-                    people[2].id === selectedPerson,
-                    100,
-                    disabled
-                  )}
-                </div>
+            </div>
+          )}
+          {/* index 2: third user*/}
```

Comment:
I'm getting myself as an opponent in people picker. I don't know if that was the intention or not but I'm assuming this made it so the number of opponents is a fixed number.

Reply:
Yup the number of opponents are hard-coded for now.